nodes/usb_cam_node.cpp

```diff
@@ -146,6 +162,7 @@ class UsbCamNode
     // create Services
     service_start_ = node_.advertiseService("start_capture", &UsbCamNode::service_start_cap, this);
     service_stop_ = node_.advertiseService("stop_capture", &UsbCamNode::service_stop_cap, this);
+    service_auto_reset_exposure_ = node_.advertiseService("reset_exposure", &UsbCamNode::service_auto_reset_exposure, this);
```

Comment:
can we remove the reset exposure param from the yaml file? i think it just makes it confusing